.github/workflows/hyperlinks.yaml

```
@@ -1,25 +1,31 @@
-name: Check Notebook Links
+name: Referential Integrity
```

To be honest, this has a different meaning. I would call it more like
```suggestion
name: Checck Hyperlinks
```

---

Agreed. See last Push.

P.S.: See https://cs.brown.edu/memex/ACM_HypertextTestbed/citationBase/D.html#[Davis%201998]